docs/js.rst

```diff
@@ -71,57 +68,57 @@ All of the mentioned 4 examples will produce a valid "action" object that can be
 Events
 ------
 
-Agile UI also offers a great way to associate your actions with certain client-side events. Those
+Agile UI also offers a great way to associate actions with certain client-side events. Those
 events can be triggered by the user or by other JavaScript code. There are several ways to bind
 `$action`.
 
-To execute actions instantly on page load, use `true` as first argument to `js()`::
+To execute actions instantly on page load, use `true` as first argument to :php:meth:`View::js()`::
 
     $view->js(
         true,
-        new jsExpression('alert([])', ['Hello world']
+        new jsExpression('alert([])', ['Hello world'])
     );
 
-You can also combine both forms together::
+You can also combine both forms::
 
     $view->js(true)->hide();
 
-
-Finally, you can specify name of JavaScript event::
+Finally, you can specify the name of the JavaScript event::
 
     $view->js('click')->hide();
 
-Agile UI also provides support for an `on` event binding. This allows you to apply events on
+Agile UI also provides support for an `on` event binding. This allows to apply events on
 multiple elements::
 
-    $buttons = new Buttons();
+    $buttons = $app->add('View');
 
-    \atk4\ui\Button::addTo($buttons, ['One']);
-    \atk4\ui\Button::addTo($buttons, ['Two']);
-    \atk4\ui\Button::addTo($buttons, ['Three']);
+    $buttons->add(['Button', 'One']);
+    $buttons->add(['Button', 'Two']);
+    $buttons->add(['Button', 'Three']);
 
```

Comment:
Please keep format for adding view like this:
````
Button::addTo($buttons, [$seed])
````
It has been update like this in all docs and we should keep it standard among documentation.

Reply:
Uh oh, sorry for that. I thought that was the "old" syntax. In fact when I tried with my install of 2.x (maybe a month old?) that didn't work ("addTo not found")...

Reply:
It was probably because the release was not synced yet with the latest core package when you tried, because addTo is part of the atk4/core package. 
Good changes in the doc. tks.

Reply:
yeah, that's only in the "develop" branch an in the ALPHA release of 2.1.

Both syntaxes work, but we want to be consistent for new users.